README.md

```diff
@@ -37,4 +53,19 @@
 |------|-------------|
 | <a name="output_resource_group_location"></a> [resource\_group\_location](#output\_resource\_group\_location) | n/a |
 | <a name="output_resource_group_name"></a> [resource\_group\_name](#output\_resource\_group\_name) | n/a |
-<!-- END_TF_DOCS -->
\ No newline at end of file
+<!-- END_TF_DOCS -->
+
+## Contributing
```

Comment:
could move this to a separate `CONTRIBUTING.md` file just copied from https://github.com/hmcts/terraform-module-postgresql-flexible